client/src/App.tsx

```diff
@@ -7,38 +7,39 @@ import Console from "./components/Console";
 import API from "./API";
 import { Stakeholders } from "./enum";
 import { toast } from './utils/toaster';
+import Layout from './components/Layout'; // Adjust the import based on your file structure
```

Comment:
should we consider the responsive on the same task? It's not responsive.
![Uploading Screen Shot 2024-12-09 at 12.11.04.png…]()